onadata/apps/api/tests/viewsets/test_entity_list_viewset.py

```diff
@@ -1739,10 +1740,10 @@ def test_download(self):
 
     def test_anonymous_user(self):
         """Anonymous user cannot download a private EntityList"""
-        # Anonymous user cannot view private EntityList
-        request = self.factory.get("/")
-        response = self.view(request, pk=self.entity_list.pk)
-        self.assertEqual(response.status_code, 404)
+        # # Anonymous user cannot view private EntityList
+        # request = self.factory.get("/")
+        # response = self.view(request, pk=self.entity_list.pk)
+        # self.assertEqual(response.status_code, 404)
```

Comment:
Does this mean that anonymouse users can now view private EntittyLists?

Reply:
@FrankApiyo Nice catch, I don't think this was intentional